cli/ingress-controller/main.go

```diff
@@ -196,9 +196,22 @@ func main() {
 		glog.Fatalf("Error creating Kong Rest client: %v", err)
 	}
 
-	root, err := kongClient.Root(context.Background())
-	if err != nil {
-		glog.Fatalf("%v", err)
+	// fix: https://github.com/Kong/kubernetes-ingress-controller/issues/761
+	var root map[string]interface{}
+	retry := 0
+	for {
+		root, err = kongClient.Root(context.Background())
+		if err != nil {
+			if retry > 5 {
+				glog.Fatalf("%v", err)
```

Comment:
Can you please rebase this change on the `next` branch instead?
A lot has changed in the next branch w.r.t. logging.